openlibrary/catalog/marc/tests/test_data/bin_expect/880_Nihon_no_chasho.json

```diff
@@ -0,0 +1,53 @@
+{
+  "publish_date": "1971",
+  "copyright_date": "1972",
+  "publish_country": "ja",
+  "languages": [
+    "jpn"
+  ],
+  "oclc_numbers": [
+    "502869803"
+  ],
+  "series": [
+    "Tōyō bunko -- 201, 206",
+    "Tōyō bunko -- 201, 206."
```

Comment:
This duplication is not great.
Looks like is is composed from multiple fields, `490` and `830` in this case

https://openlibrary.org/show-records/marc_columbia/Columbia-extract-20221130-008.mrc:340428848:1828

If it is easy, strip trailing periods (using an existing method), and remove duplicates for the `series` list.

Not really part of this feature, but it might be a simple fix.

Reply:
fixed in 93c0bd0417fbe7d051446a3771a200bea8ed5bea